src/plugins/cosmos/CosmostAssetTxHistory.tsx

```diff
@@ -0,0 +1,18 @@
+import { useParams } from 'react-router-dom'
+import { AssetHeader } from 'components/AssetHeader/AssetHeader'
+import { Main } from 'components/Layout/Main'
+import { AssetTransactionHistory } from 'components/TransactionHistory/AssetTransactionHistory'
+
+import { MatchParams } from './CosmosAsset'
+
+export const CosmosAssetTxHistory = (props: { chainId: string }) => {
```

Comment:
```suggestion
export const CosmosAssetTxHistory: React.FC<CosmosAssetTxHistoryProps> = ({ assetSubId, chainId }) => {
```

then we can clean up below

Reply:
I don't really get why you added `assetSubId` as a prop, pushed the prop refactor with `chaindId`, tell me if it looks good for you, overwise I missed something!